data/json/items/magazine/exodii.json

```diff
@@ -56,5 +56,55 @@
     "reload_time": 110,
     "flags": [ "MAG_BULKY" ],
     "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "273x44": 5 } } ]
+  },
+  {
+    "id": "ree_33_4_mag",
+    "looks_like": "stanag30",
+    "type": "MAGAZINE",
+    "name": { "str": "3.3 naval 4-round magazine" },
+    "description": "Bulky 4-round magazine for Exodii naval rifle.  Grant some amount of portability, if centner piece of metal can be named portable.",
```

Comment:
```suggestion
    "description": "Bulky 4-round magazine for Exodii naval rifle.  Grants some amount of portability, if hundredweight piece of metal can be named portable.",
```

Reply:
Isn't hundredweight is specifically 100 pounds, which is only 50 kg?

Reply:
Google translate says hundredweight is a centner, which is 100 kg.
![Screenshot_2024-11-26-13-23-18-584_com google android apps translate-edit](https://github.com/user-attachments/assets/a6e1916e-8054-4ec7-bb53-ca9ac42b384e)
![Screenshot_2024-11-26-13-22-23-356_com google android apps translate](https://github.com/user-attachments/assets/0ea2176f-4d84-4e7d-bffa-98bdfd10683a)

Reply:
You're right, it's ambiguous. Maybe we should leave it as a centner or a quintal.

Reply:
nah, i think it's okay, we'll just treat it as synonym for "very heavy"